src/app/new-page/page.component.html

```diff
@@ -4,14 +4,17 @@
   <div class="outerbox">
     <header id="toolHeader">
       <div class="header tc relative">
+        <app-sharing-modal [book]="selectedBookName"></app-sharing-modal>
+        <span class="logo">
```

Comment:
You may be able to remove this `<span>` wrapper and put the class directly on the `<a>`.

Reply:
ahh yep that worked